WMIAdventure/backend/WMIAdventure_backend/IngameUsers/businesslogic/experience/tests/test_Experience.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+
+from IngameUsers.businesslogic.experience.Experience import Experience
+from IngameUsers.businesslogic.experience.formulas import level_to_exp
+
+
+class ExperienceTestCase(unittest.TestCase):
+    def test_should_transform_exp_to_level_correctly(self):
+        expected_level = 20
+        exp = level_to_exp(expected_level) + 1
+        experience_obj = Experience(exp)
+        self.assertEqual(experience_obj.level, expected_level)
+
+    def test_should_transform_exp_to_level_correctly_first_level(self):
+        expected_level = 1
+        exp = 1
+        experience_obj = Experience(exp)
+        self.assertEqual(experience_obj.level, expected_level)
+
+    def test_should_transform_exp_to_level_correctly_fib_case(self):
+        expected_level = 5
+        exp = level_to_exp(expected_level) + 1
```

Comment:
```suggestion
        exp = level_to_exp(expected_level)
```

to się tyczy mojej sugestii z wcześniej

Reply:
Tutaj tak samo to bez znaczenia